docs/docs/eslint.md

```diff
@@ -2,7 +2,82 @@
 title: ESLint
 ---
 
-This is a stub. Help our community expand it.
+## Why ESLint
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Gatsby ships with Prettier, which is a simple, opinionated code *formatter*. [ESLint](https://eslint.org) can be both a *linter* and *formatter*, meaning you can use it to check for syntactical errors as well as formatting. Prettier will work for most sites, however if you'd like to add linting capabilities *and* highly-configurable formatting you should implement ESLint into your new Gatsby project.
+
+## How to use ESLint
+
+Here we will explore an ESLint configuration that acts like Prettier by adhering to [Standard.js](https://standardjs.com) rules. ESLint might seem intimidating at first, however it is aimed at providing a number of configurable options to make your code format fit your style. Run the following commands to remove Prettier and install ESLint.
+
+```bash
+# Remove the Prettier package
+npm rm prettier
+
+# Install ESLint and its packages
+npm install --save-dev eslint babel-eslint \
+  eslint-config-standard \
+  eslint-plugin-standard eslint-plugin-react \
+  eslint-plugin-import eslint-plugin-promise \
+```
+
+Now that we have our packages installed, remove `.prettierrc` from the root of your new Gatsby project and create a new file named `.eslintrc.js` using the commands below. 
+
+```bash
+# Remove the Prettier config file
+rm .prettierrc
+
+# Create a config file for ESLint
+touch .eslintrc.js
+```
+
+### Configuring ESLint
+
+We recommend copying our default .eslintrc.js content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options.
+
+```js
```

Comment:
@DSchau recently implemented a way to add the title of a file to the top of a code snippet. You would write it as follows:

```md
```js:title=.eslintrc.js
```